tslint.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "tslint-config-standard",
+  "rules": {
+    "trailing-comma": [true, {"multiline": "always", "singleline": "never"}]
```

Comment:
It seems like this should match the configuration in standard. Otherwise it will be very frustrating when switching between js and ts files.